core/services/ardupilot_manager/firmware_download/test_FirmwareDownload.py

```diff
@@ -0,0 +1,36 @@
+from firmware_download.FirmwareDownload import FirmwareDownload, Vehicle
+
+
+def test_static() -> None:
+    downloaded_file = FirmwareDownload._download(FirmwareDownload._manifest_remote)
+    assert downloaded_file.exists(), "Download file does not exist."
```

Comment:
this is interesting....
test_FirmwareDownload.py:6:11: E1101: Instance of 'PurePath' has no 'exists' member (no-member)
test_FirmwareDownload.py:9:11: E1101: Instance of 'PurePath' has no 'stat' member (no-member)

If this is true, shouldnt the test be failing?

Reply:
neverming, pylint is insane

Reply:
![image](https://user-images.githubusercontent.com/1215497/92005046-5b4d9d80-ed19-11ea-942c-2fd249524286.png)